modules/oauth-configuring-token-inactivity-timeout-clients.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * authentication/configuring-oauth-clients.adoc
+
+[id="oauth-token-inactivity-timeout_{context}"]
+= Configuring token inactivity timeout for an OAuth client
+
+You can configure OAuth clients to expire OAuth tokens after a set period of inactivity. By default, no token inactivity timeout is set.
+
+[NOTE]
+====
+If the token inactivity timeout is also configured in the internal OAuth server configuration, the timeout that is set in the OAuth client overrides that value.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have configured a password-based identity provider (IDP), such as HTPasswd or LDAP.
```

Comment:
It doesn't have to be a password based IDP. Any IDP should work with this feature.

Reply:
Updated.